fixit/rules/import_constraints.py

```diff
@@ -314,6 +390,56 @@ class ImportConstraintsRule(CstLintRule):
             ),
             filename="dir_1/dir_2/file.py",
         ),
+        # Deny global
+        Invalid(
+            """
+            import common
+            """,
+            config=_gen_testcase_config({"dir": {"rules": [["*", "deny global"]]}}),
+            filename="dir/file.py",
+        ),
+        # Deny local
+        Invalid(
+            """
+            def local_scope():
+                import common
+            """,
+            config=_gen_testcase_config({"dir": {"rules": [["*", "deny local"]]}}),
+            filename="dir/file.py",
+        ),
+        # Allow global
+        Invalid(
+            """
+            def local_scope():
+                import common
+            """,
+            config=_gen_testcase_config({"dir": {"rules": [["*", "allow global"]]}}),
+            filename="dir/file.py",
+        ),
+        # Allow local
+        Invalid(
+            """
+            import common
+            """,
+            config=_gen_testcase_config({"dir": {"rules": [["*", "allow local"]]}}),
+            filename="dir/file.py",
+        ),
+        # Custom message
+        Invalid(
+            """
+            import common
+            """,
+            config=_gen_testcase_config(
+                {
+                    "dir": {
+                        "rules": [["*", "deny"]],
+                        "message": "'{imported}' cannot be imported from '{current_file}'",
```

Comment:
I like the idea to allow custom message.
Can you update the docstring of ImportConstraintsRule and explain how to use the message?
We should explain that `imported` and `current_file` are two variables that can be embedded in the message.